### ISSUE TITLE
Changes in NVDA settings should be saved in the normal configuration unless the user actively chooses to save the change in a separate profile.

### ISSUE THREAD

**Issue by rperez030** [nvaccess/nvda#11256]
### Is your feature request related to a problem? Please describe.
If I change an NVDA setting on the fly such as the speaking of type characters, screen curtain or synth setting while a profile is active, the change will revert back to normal as soon as I leave the application that triggered the profile. In addition, the change I made on the fly will be saved to the active profile and will be loaded as soon as I set focus to the application that triggers it. This is particularly frustrating with screen curtain which gets disabled as soon as I Alt tab to an application that loads a profile. While working, for example, I constantly play with the speech volume, especially if I need to make a call, so I end up having different speech volumes in different applications. My solution right now is to manually edit the .ini files and erase unwanted settings from each profile from time to time.
### Describe the solution you'd like
Changes made on the fly using keyboard shortcuts such as the speaking of typed characters, screen curtain or synth settings should be saved in the default configuration by default. This would resolve the immediate problem of Inadvertently saving settings in an active profile. In addition, the following steps can be taken to give the user more control over where settings are saved:
- Add a dropdown menu in the Settings dialog for the user to choose where the changes should be saved (E.G. normal configuration, Outlook, etcetera).
- Add a "reset to default" button per each setting category which will cause that particular section to be deleted in the .ini file for the corresponding profile.

### Describe alternatives you've considered
All the other alternatives I can think of are more complex. For example, implementing a profile editor where the user can select which setting categories should be saved in a particular profile. After that, if a changed is made on the fly while a profile is active and the corresponding category is present in the .ini file the change will be saved in the profile, otherwise it will be saved in the default configuration. This editor could also allow the removal of specific sections from an individual profile as well as the import / export of user profiles.
### Additional context
I am running NVDA version 2020.1.

**Comment by Adriani90** [comment]
Actually the solution described here is not ideal. When you have more than 50 profiles, it does not make sense to choose via a combo box in which profile the setting should be applied. Instead, I propose a shortcut to apply profile specific settings to all profiles or to default profile, but not to copy settings to another profile. Because then it gets really complex.
So the two options (apply settings from this profile to default profile)
and (apply this profile settings to all profiles)
should solve most problems.

However, these concerns are already addressed in #10476 and #10156 as well as #4472. I am closing this issue in favor of referenced issues already in place.
